tests/integration/helpers.bash

```diff
@@ -55,10 +55,11 @@ function setup_test() {
             exit 1
             ;;
     esac
-    cd "$TEST_BUNDLE"
+    cd "$TEST_BUNDLE" || exit
 }
 
 function config_mod () {
+    # shellcheck disable=SC2002
```

Comment:
Hmm, what is the particular issue here?

Reply:
It says unnecessary cat and to use a redirect, I did a quick search and found another repo doing the exact same thing we do and just ignoring it. We could probably open an issue on shellcheck about it but haven't looked that route yet.

Reply:
Hmm that's a valid point though, can we do `jq "$@" > config.json.new < config.json` ?

Reply:
this is fixed with latest commit. I need to read up on shell redirects :P